moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummary.js

```diff
@@ -60,159 +66,183 @@ const ProjectSummary = ({
+                      initialValue={
+                        data?.moped_project[0]?.moped_public_process_statuses
+                      }
+                      optionList={data?.moped_public_process_statuses ?? []}
+                      updateMutation={PROJECT_UPDATE_PUBLIC_PROCESS}
+                      tooltipText="Current public phase of a project"
+                      projectId={projectId}
+                      loading={loading}
+                      data={data}
+                      refetch={refetch}
+                      handleSnackbar={handleSnackbar}
+                    />
+                  </Grid>
+                  <Grid item xs={12}>
+                    <ProjectSummaryProjectWebsite
+                      projectId={projectId}
+                      loading={loading}
+                      data={data}
+                      refetch={refetch}
+                      handleSnackbar={handleSnackbar}
+                    />
+                  </Grid>
+                  <Grid item xs={12}>
+                    <ProjectSummaryInterimID
+                      projectId={projectId}
+                      loading={loading}
+                      data={data}
+                      refetch={refetch}
+                      handleSnackbar={handleSnackbar}
+                    />
+                  </Grid>
+                  <Grid item xs={12}>
+                    <ProjectSummaryProjectECapris
+                      projectId={projectId}
+                      loading={loading}
+                      data={data}
+                      refetch={refetch}
+                      handleSnackbar={handleSnackbar}
+                    />
+                  </Grid>
+                  <Grid item xs={12}>
+                    <ProjectSummaryDataTrackerSignals
+                      project={data?.moped_project?.[0]}
+                    />
+                  </Grid>
+                  <Grid item xs={12}>
+                    <ProjectSummaryWorkOrders
+                      project={data?.moped_project?.[0]}
+                    />
+                  </Grid>
+                </Grid>
+              </CardContent>
+            </Card>
           </Grid>
           <Grid item xs={12} md={6}>
             <Grid container spacing={2}>
               <Grid item xs={12}>
-                <ProjectSummaryMap data={data} />
-              </Grid>
-              <Grid item xs={12}>
-                <ProjectSummaryCouncilDistricts
-                  projectGeography={data.project_geography}
-                  childProjectGeography={childProjectGeography}
-                />
+                <Card sx={{ height: "fit-content" }}>
+                  <CardContent>
+                    <Typography variant="h2" color="primary" sx={{ mb: 3 }}>
+                      Map
+                    </Typography>
+                    <Grid container spacing={2}>
+                      <Grid item xs={12}>
+                        <ProjectSummaryMap data={data} />
```

Comment:
When I was testing, I noticed that `ProjectSummaryMap` renders `ProjectSummaryMapFallback` if there are no components in a project which creates a double card. I think it could be a quick fix to remove the card wrapper in `ProjectSummaryMapFallback` and recycle the sx value in the surrounding `Box`. Could you update or I could also make a spin-off snackoo?

It is only noticeable on new projects.

<img width="699" height="540" alt="Image" src="https://github.com/user-attachments/assets/89c9d502-7751-4c97-a6d6-f38a4b2b95cb" />